app/views/admin/submissions/_notes_section.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if notes && notes.length > 0 %>
+  <div class='notes-section'>
+    <div class='notes-section-title'>
+      Notes
+    </div>
+    <%= render partial: 'admin/submissions/note', collection: @notes %>
```

Comment:
I feel like this file could have gone inline in `show.html.erb`, but I was copying from the line above.